src/pages/user/user.jsx

```diff
@@ -0,0 +1,129 @@
+import userStyle from "./user.module.scss";
+import emailImg from "./images/email.svg";
+import phoneImg from "./images/phone.svg";
+import passwordImg from "./images/password.svg";
+import profileImg from "./images/profile.svg";
+import nextImg from "./images/next.svg";
```

Comment:
Move all the images to elements so that they can be reused

Reply:
Done